### DISCUSSION TITLE
How to put svg icon into button or use as button

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#79] [Web] [answered]
Hallo zusammen, 

ich habe folgendes Problem: ich kann meinen importierten svg icon von Figma nicht als Button abbilden. 
ich habe folgendes probiert: ich habe das svg als Imagine über next/image importiert was nicht funktioniert hat und ich habe versucht das svg als element zu importieren beides funktioniert irgendwie nicht. 
siehe Bilder hat jemand eine Idee? 
![Screenshot 2022-08-17 at 13 42 12](https://user-images.githubusercontent.com/102238180/185110764-1d85915c-a8fc-4d26-a8e8-b0b1977d8986.png)
![Screenshot 2022-08-17 at 13 43 34](https://user-images.githubusercontent.com/102238180/185110785-828e60ac-d328-4cb3-8eaa-a42e20284b6d.png)
 Freue mich über Ratschläge

**Comment by alexclaes** [comment, marked as answer]
Hallo Philip,

grundsätzlich zeigt dein Code einen richtigen Ansatz. 

Du verstößt allerdings gegen eine ganz grundlegende Regel von React Komponenten. Lese bitte noch einmal in der Doku nach:
https://reactwithhooks.netlify.app/docs/components-and-props.html#rendering-a-component

Wenn du mit der Maus auf die rote markierten Stellen im Code fährst, zeigt VS Code dir sicher auch eine hilfreiche Fehlerbeschreibugn an.

Bedenke bitte, dass klickbare Elemente den `<button>`Tag nutzen müssen, um zugänglich zu sein

**Reply by PhilipZi** [reply]
vielen Dank für die Hilfe und die Ressource, ich habe versucht es an Hand deines Beispiels um zusetzten allerdings hat es nicht funktioniert. Hier ein Bild von dem update:
![Screenshot 2022-08-17 at 15 13 49](https://user-images.githubusercontent.com/102238180/185143389-5c9a6fde-2731-4c7c-95c1-76bebe942414.png)

**Reply by PhilipZi** [reply]
auch diese Variante funktioniert leider nicht (siehe Bild)
![Screenshot 2022-08-17 at 15 20 08](https://user-images.githubusercontent.com/102238180/185144545-ec0f45f7-827e-4bfd-8f1b-665a85bd6e5d.png)

**Reply by PhilipZi** [reply]
here my github
https://github.com/PhilipZi/capstone-project/blob/edit-timer-card/src/components/timer/timer1.js

**Reply by F-Kirchhoff** [reply]
Please take a look at the `naming conventions` for react components.

**Reply by PhilipZi** [reply]
Das war es DANKE! habs jetzt gross geschrieben! Danke Felix und happy to hear from you.
